### PR TITLE
feat(wyoming-piper) Add wyoming-piper

### PR DESCRIPTION
**Pull request by fenichelar**
**Description**

⚒️ Fixes  # 

Add [wyoming-piper](https://github.com/home-assistant/addons/blob/master/piper/DOCS.md) (TTS ) for Home Assistant.

**⚙️ Type of change**

- [x] ⚙️ Feature/App addition
- [ ] 🪛 Bugfix
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] 🔃 Refactor of current code

**🧪 How Has This Been Tested?**

**📃 Notes:**

**✔️ Checklist:**

- [x] ⚖️ My code follows the style guidelines of this project
- [x] 👀 I have performed a self-review of my own code
- [ ] #️⃣ I have commented my code, particularly in hard-to-understand areas
- [ ] 📄 I have made corresponding changes to the documentation
- [ ] ⚠️ My changes generate no new warnings
- [ ] 🧪 I have added tests to this description that prove my fix is effective or that my feature works
- [ ] ⬆️ I increased versions for any altered app according to semantic versioning

**➕ App addition**

If this PR is an app addition please make sure you have done the following.

- [x] 🪞 I have opened a PR on [truecharts/containers](https://github.com/truecharts/containers) adding the container to TrueCharts mirror repo.
- [x] 🖼️ I have added an icon in the Chart's root directory called `icon.png`

---

_Please don't blindly check all the boxes. Read them and only check those that apply.
Those checkboxes are there for the reviewer to see what is this all about and
the status of this PR with a quick glance._

### CODE REVIEW

File: mirror/wyoming-piper/PLATFORM [hunk 1]
```diff
@@ -0,0 +1 @@
+linuxamd64
```
Comment (stavros-k) [inline mirror/wyoming-piper/PLATFORM:1]:
```suggestion
linux/amd64
```

File: mirror/wyoming-piper/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,10 @@
+FROM rhasspy/wyoming-piper:latest@sha256:e62fa006b6fccda2f1be2f1fd8229bc6c1139b8adeb2e6d479f411e628bb1c90
```
Comment (stavros-k) [inline mirror/wyoming-piper/Dockerfile:1]: Use version tag please. 
Not latest

Comment (stavros-k) [inline mirror/wyoming-piper/Dockerfile:1]:
```suggestion
FROM rhasspy/wyoming-piper:0.0.2@sha256:e62fa006b6fccda2f1be2f1fd8229bc6c1139b8adeb2e6d479f411e628bb1c90
```

Comment (fenichelar) [reply mirror/wyoming-piper/Dockerfile]: The version tag didn't exist when I made this PR. It looks like it was created 2 days ago.
FYI, the digest changed with the newly released version.

**Pull request merged**